src/snowflake/snowpark/mock/functions.py

```diff
@@ -137,7 +139,7 @@ def mock_avg(column: ColumnEmulator) -> ColumnEmulator:
             cnt += 1
     # round to 5 according to snowflake spec
     ret = (
-        ColumnEmulator(data=[round((ret / cnt), 5)])
+        ColumnEmulator(data=[round((ret / cnt), 3)])
```

Comment:
there are other places in this module where we round to 5 (I did this previously)
so do we also need to update other mock implementations?

Reply:
Yea, I will address these altogether in a separate PR.